### ISSUE TITLE
Duplicate module name: glob

### ISSUE THREAD

**Issue by katgironpe** [react-community/create-react-native-app#95]
When running `npm start`, I get this:

```
jest-haste-map: @providesModule naming collision:
  Duplicate module name: glob
  Paths: /Users/kath/mobile/my-project/node_modules/react-native/node_modules/rimraf/node_modules/glob/package.json collides with /Users/kath/mobile/my-project/node_modules/react-native/node_modules/glob/package.json

This warning is caused by a @providesModule declaration with the same name across two different files.
```

**Comment by dawnvan** [comment]
delete rimraf/node_modules/glob

**Comment by dikaiosune** [comment]
Have you run `npm link` or `yarn link` for any of your app's dependencies?

**Comment by katgironpe** [comment]
@dikaiosune I haven't

**Comment by dikaiosune** [comment]
Interesting. Maybe try deleting node_modules and reinstalling? If that
doesn't work can you paste your package json?

Adam Perry

**Comment by dikaiosune** [comment]
Hi! If you're still having problems with this please reopen the issue and I'll see if I can reproduce.

**Comment by luferogo** [comment]
Hello, I am having the same issue. 
This is my package json:
`{
  "name": "MiColonia.Mobile.Client",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "react-native-scripts": "0.0.26",
    "jest-expo": "^0.3.0",
    "react-test-renderer": "~15.4.1"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js --watch"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "expo": "^15.1.0",
    "react": "~15.4.0",
    "react-native": "0.42.3"
  }
}`

The steps I followed to get my app running were:
1. Upgrade NodeJS from version 4.x to 6.x
2. Installed npm install -g create-react-native-app
3. Installed XDE

Any idea for solving this?

**Comment by dikaiosune** [comment]
If you're running the project in XDE, you'll need to use the Expo fork of RN. If you need help with XDE, I'd recommend searching for your issue on the [Expo forums](https://forums.expo.io/).

**Comment by pmdarrow** [comment]
I'm experiencing this error with a brand new project after running
```
npm install -g create-react-native-app
create-react-native-app my-app
cd my-app
npm start
```
I'm using node v6.10.2 and npm 3.10.10.

**Comment by pmdarrow** [comment]
After upgrading to node 7.8.0 and switching to yarn 0.22.0, I no longer experience this problem.

**Comment by yash2code** [comment]
same issue

**Comment by sixertoy** [comment]
same issue, resolved with nvm + node 7.8.0 + yarn v0.23.2

**Comment by SoldierCorp** [comment]
Fixed with: Node 8.1.0 + yarn 0.23.4
